Core/src/Surfaces/LineSurface.cpp

```diff
@@ -110,16 +104,15 @@ std::string Acts::LineSurface::name() const {
 Acts::RotationMatrix3 Acts::LineSurface::referenceFrame(
     const GeometryContext& gctx, const Vector3& /*position*/,
     const Vector3& direction) const {
+  Vector3 measZ0 = transform(gctx).rotation() * Vector3::UnitZ();
+  Vector3 measD0 = measZ0.cross(direction).normalized();
+  Vector3 measDepth = measD0.cross(measZ0);
```

Comment:
```suggestion
  Vector3 measDepth = direction.normalized();
  Vector3 meadD0 = measZ0.cross(measDepth);
```

Reply:
The variable names are not obvious for me, but this might be due to my ignorance

Reply:
I also don't like them but I had no better idea. maybe `unitD0` `unitZ0` `unitNormal` is better?

I wanted to make the same change but `direction` is actually not pointing in the same way as `measDepth` in the general case.

Reply:
Maybe something like `dirD0` etc.?

Hm this I don't quite understand... Does that mean that `direction` and `measZ0` are not orthogonal to each other in the general case?